pom.xml

```diff
@@ -55,6 +55,7 @@
     <modules>
         <module>timachine-core</module>
         <module>timachine-maven-plugin</module>
+        <module>timachine-maven-plugin-test</module>
```

Comment:
Even though we don't use unit test, I recommend you to add as a sub module. It will help you to bump up versions in `pom.xml` automatically (by `maven-release-plugin`). It also reminds us about changes in plugin configuration.

> If you have some good way to test plugin usage automatically, please tell me, thanks!

It is also possible, I use this method [in my FindBugs plugin](https://github.com/KengoTODA/findbugs-slf4j/tree/master/test-case).